lib/hyku_knapsack/engine.rb

```diff
@@ -58,13 +54,7 @@ def self.load_translations!
         Rails.configuration.cache_classes ? require(c) : load(c)
       end
 
-      # By default plain text files are not processed for text extraction.  In adding
-      # Adventist::TextFileTextExtractionService to the beginning of the services array we are
-      # enabling text extraction from plain text files.
-      #
-      # https://github.com/scientist-softserv/adventist-dl/blob/97bd05946345926b2b6c706bd90e183a9d78e8ef/config/application.rb#L68-L73
       Hyrax::DerivativeService.services = [
-        Adventist::TextFileTextExtractionService,
```

Comment:
I suspect the TextFileTextExtractionService is actually something we generally want.  It so happens to be specifically named.